generated_code/acoustic.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+##
+# @file
+# This file is part of SeisSol.
+#
+# @author Carsten Uphoff (c.uphoff AT tum.de, http://www5.in.tum.de/wiki/index.php/Carsten_Uphoff,_M.Sc.)
+#
+# @section LICENSE
+# Copyright (c) 2016-2018, SeisSol Group
```

Comment:
Change year to 2024 in the end. Or better adapt license from one of the recent license changes David made.